vllm/engine/arg_utils.py

```diff
@@ -320,6 +322,10 @@ def add_cli_args(
                             'None, we assume the model weights are not '
                             'quantized and use `dtype` to determine the data '
                             'type of the weights.')
+        parser.add_argument('--rope-scaling',
+                            default=None,
+                            type=json.loads,
+                            help='RoPE scaling configuration')
```

Comment:
Can you share an example of what the CLI format is like for this arg in the help function?